apm-sniffer/apm-sdk-plugin/servicecomb-plugin/servicecomb-java-chassis-2.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/servicecomb/v2/ProducerOperationHandlerInterceptor.java

```diff
@@ -42,7 +42,12 @@ public void beforeMethod(EnhancedInstance objInst, Method method, Object[] allAr
         CarrierItem next = contextCarrier.items();
         while (next.hasNext()) {
             next = next.next();
-            next.setHeadValue(invocation.getContext().get(next.getHeadKey()));
+            String headKey = next.getHeadKey();
+            if (invocation.getContext().containsKey(headKey)) {
+                next.setHeadValue(invocation.getContext().get(headKey));
+            } else {
+                next.setHeadValue(invocation.getRequestEx().getHeader(headKey));
```

Comment:
Please add a comment for this fallback solution.

Reply:
Add not cse invoke scenario. I have test it was ok.

Reply:
I think the request to code comments